myParser.py

```diff
@@ -79,15 +91,13 @@ def parse():
         "-m",
         type=int,
         required=True,
-        help="The total fare for one person should be under this amount (in dollars).")
+        help="The total fare for one person should be under this amount (in dollars or points).")
 
     parser.add_argument(
```

Comment:
Can you edit the Usage section of the README to reflect the update use of this command?

Reply:
Yep